roles/magnum/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: magnum user
+  user: name=magnum comment=magnum shell=/bin/false system=yes
+        home=/nonexistent createhome=no
+
+- name: magnum config dir
+  file: dest=/etc/magnum state=directory
+
+- name: magnum log dir
+  file: dest=/var/log/magnum state=directory mode=0755 owner=magnum
+        group=magnum
+
+- name: install magnum service
+  upstart_service: name={{ item }} user=magnum cmd=/usr/local/bin/{{ item }}
+                   config_dirs=/etc/magnum
+  with_items:
+    - magnum-api
+    - magnum-conductor
+
+- name: configure magnum
+  template: src={{ item }} dest=/etc/magnum/ mode=0644
+  with_fileglob: ../templates/etc/magnum/*
+  notify:
+    - restart magnum services
+
+- name: stop magnum service before db sync
+  service: name={{ item }} state=stopped
+  with_items:
+    - magnum-api
+    - magnum-conductor
+  when: database_create.changed or force_sync|default('false')|bool
+
+- name: sync magnum database
+  command: magnum-db-manage upgrade
+  when: database_create.changed or force_sync|default('false')|bool
+  run_once: true
+  changed_when: true
+  notify: restart magnum services
+  # we want this to always be changed so that it can notify the service restart
+
+- meta: flush_handlers
+
+  service: name={{ item }} state=started
+  with_items:
+    - magnum-api
+    - magnum-conductor
+
+- name: permit access to magnum
+  ufw: rule=allow to_port={{ item }} proto=tcp
+  with_items:
+    - 9511
+
+#- include: monitoring.yml tags=monitoring,common
+
+#- include: logging.yml
+#  tags:
+#    - logrotate
+#    - logging
```

Comment:
yeah ..  basically needing to be implemented ... so commented out.         this PR is not "feature complete ready to run in production"  it's "MVP ... it installs and works".     We can feature flag it out of CI and other installs until get it better.